subsys/partition_manager/flash_map_partition_manager.c

```diff
@@ -49,3 +50,10 @@ const struct flash_area default_flash_map[] = {
 
 const int flash_map_entries = ARRAY_SIZE(default_flash_map);
 const struct flash_area *flash_map = default_flash_map;
+
+#else
+
+const int flash_map_entries = 0;
```

Comment:
`const int`?

Reply:
Nope. Because https://github.com/nrfconnect/sdk-zephyr/blob/main/subsys/storage/flash_map/flash_map_default.c and https://github.com/nrfconnect/sdk-zephyr/blob/main/subsys/storage/flash_map/flash_map.c#L22.